pkg/core/bootstrap/bootstrap.go

```diff
@@ -125,6 +126,9 @@ func buildRuntime(appCtx context.Context, cfg kuma_cp.Config) (core_runtime.Runt
 		))
 	}
 
+	// The setting should be removed, and there is no easy way to set it without breaking most of the code
+	mesh_proto.EnableLocalhostInboundClusters = builder.Config().Defaults.EnableLocalhostInboundClusters
```

Comment:
Would it be possible to add this setting to the `Dataplane` proto and then, based on the CP setting, setting the field when a `Dataplane` is created?

Reply:
It's possible but later might be problematic. This field should be removed in the future if we introduce a new field in the meanwhile why might need to keep the order and the field.